### ISSUE TITLE
Overlapping Window Positions

### ISSUE THREAD

**Issue by Tealk** [domferr/tilingshell#133]
**Describe the feature you'd like**  
It would be nice to have the ability to allow overlapping window positions in the tiling manager. This feature would be especially useful for users with ultra-wide (UW) monitors, as it would provide more flexibility in managing screen real estate and positioning windows according to specific needs.
I'm always frustrated when I can't overlap windows in the tiling manager, especially on ultra-wide monitors. The fixed grid layout restricts efficient use of the screen, and overlapping windows would allow for more efficient multitasking and better use of the available space.

**Comment by domferr** [comment]
Hey, thank you for sharing this idea. Tiling Shell supports this behind the scenes. However the editor doesn't, hope to update it in the future!

**Comment by kubijo** [comment]
I just wanted to create the same ask for the same reason... idea being that you want your windows accessible to click-focus bud using the most available space...

It's great that it is stacked up to happen @domferr!

**Comment by butterdori** [comment]
Yes, it would be nice to have this. Powertoys supports this, and I use it to have a panel that covers about 80% of the screen, placed in the middle. This is used to automatically set window sizes of windows I don't want maximized, but still larger and wider than left/right halves of the screen.

**Comment by RobMayer** [comment]
chiming in with another wish for this to be a thing.
